Tests/docker_native_image_config.json

```diff
@@ -60,6 +60,37 @@
             "chromium",
             "tesseract"
          ]
+      },
+      "native:candidate":{
+         "supported_docker_images":[
+            "python3",
+            "python3-deb",
+            "python3-ubi",
+            "py3-tools",
+            "py3-tools-ubi",
+            "crypto",
+            "readpdf",
+            "parse-emails",
+            "docxpy",
+            "sklearn",
+            "pandas",
+            "ippysocks-py3",
+            "oauthlib",
+            "unzip",
+            "py3ews",
+            "taxii2",
+            "pan-os-python",
+            "slackv3",
+            "google-api-py3",
+            "boto3py3",
+            "pyjwt3",
+            "joe-security",
+            "slack",
+            "office-utils",
+            "chromium",
+            "tesseract"
+         ],
+         "docker_ref": "demisto/py3-native:8.2.0.49457"
```

Comment:
when we merge lets change this to >= the current 8.2 image

Reply:
Updated this image to the last image the test-native-candidate ran on
https://code.pan.run/xsoar/content/-/pipelines/4874846